cli/command/environment_test.go

```diff
@@ -0,0 +1,145 @@
+package command
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/cli/printer"
+	"github.com/quintilesims/layer0/cli/resolver/mock_resolver"
+	"github.com/quintilesims/layer0/client/mock_client"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateEnvironment(t *testing.T) {
+	client, _, envCommand, ctrl := initEnvCommandTest(t)
+	defer ctrl.Finish()
+
+	userData := "user_data"
```

Comment:
we probably don't need this overly-explicit behavior outside of the `provider/aws` package